docs/repositories.rst

```diff
@@ -78,3 +78,26 @@ Documentation
      - Description
    * - https://github.com/RackHD/docs
      - The RackHD documentation as published to http://rackhd.readthedocs.org/en/latest/.
+
+
+Repositories Status
+^^^^^^^^^^^^^^^^^
```

Comment:
yep, these are the "tags and badges" that we have enabled on the README's for each of the repos - and show build status (travisCI), test coverage (coveralls), static analysisresults (code climate)